ionoscloud/resource_dbaas_mongodb_cluster.go

```diff
@@ -54,7 +56,6 @@ func resourceDbaasMongoDBCluster() *schema.Resource {
 				Type:             schema.TypeString,
 				Description:      "The MongoDB version of your cluster. Update forces cluster re-creation.",
```

Comment:
update description if it no longer requires re-creation.

Reply:
done